sdk/core/amqp/package.json

```diff
@@ -102,7 +103,7 @@
     "prepack": "npm i && npm run build:node",
     "pack": "npm pack 2>&1"
   },
-  "homepage": "https://github.com/Azure/azure-sdk-for-js",
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/core/amqp",
```

Comment:
I don't necessary like this linking to the existing path in the repo as we might change the location in the future. Is there a reason you change this?

Reply:
That is because of the guidelines : https://azuresdkspecs.z5.web.core.windows.net/TypeScriptSpec.html#ts-package-json

Reply:
Other packages are using this guideline, should we update the guidelines?

Reply:
@bterlson for this guideline would it be acceptable if the link points to the root of the repo or the package index like packages.md (https://github.com/Azure/azure-sdk-for-js/blob/master/packages.md)  in the root of the repo? I ask mainly because if we link directly to paths and we move them like we just did those links break. So it really depends on our SLA for keeping those links from breaking.

Reply:
To me the high order bit is how are users on npmjs.com able to find our source code. I believe most of the time they'll just click homepage. In the case of repo restructuring, this will be problematic only for old packages and packages that haven't been updated to reflect the new location. To me the wins here outweigh the downside, especially considering we might move repos, move pack to uni-repos, packages.md may change, or whathaveyou.

Reply:
Agree, I think this change helps users find the source code easily.

Reply:
OK I'm fine with this so long as we are willing to accept that if we need to move the location in the future the older package homepage links may break and we cannot use that as a reason to not make the move.

Reply:
@ramya0820 
Can you update the homepage link to point to the release tag instead so that it will be permanent? https://github.com/Azure/azure-sdk-for-js/tree/%40azure/amqp-common_1.0.0-preview.4/sdk/core/amqp-common

Reply:
Got it, thanks - I've updated the references @KarishmaGhiya @weshaggard